src/badguy/corrupted_granito.cpp

```diff
@@ -1,4 +1,4 @@
-//  SkullyHop - A Hopping Skull
+//  Corrupted Granito - A "Evil" Granito
 //  Copyright (C) 2006 Christoph Sommer <christoph.sommer@2006.expires.deltadevelopment.de>
```

Comment:
Since you changed name of this file and made changes you should add yourself to the license header (same for the hpp file).